src/client/mod.rs

```diff
@@ -143,6 +143,58 @@ impl<T: Transport, L: LoginCredentials> TwitchIRCClient<T, L> {
             .await
     }
 
+    /// Ban a user with a optional reason in the given Twitch channel.
+    pub async fn ban(
+        &self,
+        channel_login: String,
+        user: String,
+        reason: Option<String>,
+    ) -> Result<(), Error<T, L>> {
+        self.privmsg(
+            channel_login,
+            format!("/ban {} {}", user, reason.unwrap_or_default()),
```

Comment:
If the `reason` is `None` the ban command would include a trailing space. Can you rewrite it so the extra space is not included at the end if `reason` is `None`?